PhysicsTools/PatAlgos/python/producersLayer1/ootPhotonProducer_cff.py

```diff
@@ -49,3 +49,44 @@
     )
 
 makePatOOTPhotons = cms.Sequence(makePatOOTPhotonsTask)
+
+## For legacy reprocessing
+## Standard Sequences
+from RecoJets.Configuration.CaloTowersES_cfi import *
+from RecoLocalCalo.EcalRecAlgos.EcalSeverityLevelESProducer_cfi import *
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitPS_cfi import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterPS_cfi import *
+## OOT Sequences
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitOOTECAL_cff import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterOOTECALUncorrected_cff import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterOOTECAL_cff import *
+from RecoEcal.EgammaClusterProducers.particleFlowSuperClusterOOTECAL_cff import *
+from RecoEgamma.EgammaPhotonProducers.ootPhotonCore_cff import *
+from RecoEgamma.EgammaPhotonProducers.ootPhotons_cff import *
+from RecoEgamma.EgammaIsolationAlgos.pfClusterIsolation_cfi import *
+
+_makePatOOTPhotonsTask = makePatOOTPhotonsTask.copy()
+_makePatOOTPhotonsTask.add(
+    CaloTowerConstituentsMapBuilder,
+    essourceEcalSev,
+    ecalSeverityLevel,
+    particleFlowRecHitPS,
+    particleFlowClusterPS,
+    particleFlowRecHitOOTECAL,
+    particleFlowClusterOOTECALUncorrected,
+    particleFlowClusterOOTECAL,
+    particleFlowSuperClusterOOTECAL,
+    ootPhotonCore,
+    ootPhotons,
+    ootPhotonEcalPFClusterIsolationProducer
```

Comment:
if I'm not mistaken, this is a re-definition of the ootPhotonSequence as a taks.
This should be moved to RecoEgamma/EgammaPhotonProducers/python/ootPhotonSequence_cff.py
and whatever upstream files from that ones. In there please define a task and then initialize the necessary cms.Sequence 

Eventually, updates in this file should look like [modulo corrections for typos I may have below]
```python
from RecoEgamma.EgammaPhotonProducers.ootPhotonSequence_cff import ootPhotonTask

+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
run2_miniAOD_80XLegacy.toReplaceWith(makePatOOTPhotonsTask, cms.Task(makePatOOTPhotons.copy(),ootPhotonTask) )

run2_miniAOD_80XLegacy.toModify(patOOTPhotons, hcalPFClusterIsoMap = "")
```